microservices/scraper-ngouganda/scraper.py

```diff
@@ -0,0 +1,41 @@
+from bs4 import BeautifulSoup
+import requests
+import json
+
+website = "https://informationcradle.com/africa/list-of-ngos-in-uganda/"
+
+def get_page_data():
+    # Specify url to scrape from
+    ret = []
+    # options = set(['Name', 'P.O. Box', 'Website', 'Description', 'Category', 'Physical Address', 'Telephone', 'E-mail', 'Contact Person', ])
+    target_url = requests.get(website)
+    page_data = BeautifulSoup(target_url.content, "html.parser")
+    contents = page_data.find('div', {'class': 'entry-content'})
+    contents = contents.find_all('p')
+    for ngo in contents:
+        d = {}
+        ngo = str(ngo).split('<br/>\n')
+        for line in ngo:
```

Comment:
cam we split the line on spaces or something so that we aren't indexing characters on a line. It seems like thsi can go wrong really fast.